actioncable/test/javascript/src/unit/subscription_test.js

```diff
@@ -0,0 +1,58 @@
+import consumerTest from "../test_helpers/consumer_test_helper"
+
+const {module} = QUnit
+
+module("ActionCable.Subscription", function() {
+  consumerTest("#initialized callback", ({server, consumer, assert, done}) =>
```

Comment:
Any reason for using `=>` here and `function` on the line above? Can we use `=>` everywhere?